packages/ast-spec/src/declaration/ClassDeclaration/fixtures/implementsMany/snapshots/7-AST-Alignment-AST.shot

```diff
@@ -0,0 +1,115 @@
+"Snapshot Diff:
+- TSESTree
++ Babel
+
+  Program {
+    type: 'Program',
+    body: Array [
+      ClassDeclaration {
+        type: 'ClassDeclaration',
+        body: ClassBody {
+          type: 'ClassBody',
+          body: Array [],
+
+          range: [46, 48],
+          loc: {
+            start: { column: 46, line: 1 },
+            end: { column: 48, line: 1 },
+          },
+        },
+        id: Identifier {
+          type: 'Identifier',
+          name: 'Foo',
+
+          range: [6, 9],
+          loc: {
+            start: { column: 6, line: 1 },
+            end: { column: 9, line: 1 },
+          },
+        },
+        implements: Array [
+-         TSClassImplements {
+-           type: 'TSClassImplements',
++         TSExpressionWithTypeArguments {
++           type: 'TSExpressionWithTypeArguments',
+            expression: Identifier {
+              type: 'Identifier',
+              name: 'Object',
+
+              range: [21, 27],
+              loc: {
+                start: { column: 21, line: 1 },
+                end: { column: 27, line: 1 },
+              },
+            },
+
+            range: [21, 27],
+            loc: {
+              start: { column: 21, line: 1 },
+              end: { column: 27, line: 1 },
+            },
+          },
+-         TSClassImplements {
+-           type: 'TSClassImplements',
++         TSExpressionWithTypeArguments {
++           type: 'TSExpressionWithTypeArguments',
+            expression: Identifier {
+              type: 'Identifier',
+              name: 'Function',
+
+              range: [29, 37],
+              loc: {
+                start: { column: 29, line: 1 },
+                end: { column: 37, line: 1 },
+              },
+            },
+
+            range: [29, 37],
+            loc: {
+              start: { column: 29, line: 1 },
+              end: { column: 37, line: 1 },
+            },
+          },
+-         TSClassImplements {
+-           type: 'TSClassImplements',
++         TSExpressionWithTypeArguments {
++           type: 'TSExpressionWithTypeArguments',
+            expression: Identifier {
+              type: 'Identifier',
+              name: 'RegExp',
+
+              range: [39, 45],
+              loc: {
+                start: { column: 39, line: 1 },
+                end: { column: 45, line: 1 },
+              },
+            },
+
+            range: [39, 45],
+            loc: {
+              start: { column: 39, line: 1 },
+              end: { column: 45, line: 1 },
+            },
+          },
+        ],
+        superClass: null,
+
+        range: [0, 48],
+        loc: {
+          start: { column: 0, line: 1 },
+          end: { column: 48, line: 1 },
+        },
+      },
+    ],
+    sourceType: 'script',
+
+    range: [0, 49],
+    loc: {
+      start: { column: 0, line: 1 },
+      end: { column: 0, line: 2 },
+    },
+  }"
+`;
```

Comment:
example of an AST misalignment.

[**Rendered in GH**](https://github.com/typescript-eslint/typescript-eslint/blob/63f5b1ab8a27cb0daf12a325988ffe674c3267be/packages/ast-spec/src/declaration/ClassDeclaration/fixtures/implementsMany/snapshots/7-AST-Alignment-AST.shot)